timeboost-sequencer/src/config.rs

```diff
@@ -51,8 +47,11 @@ pub struct SequencerConfig {
     /// Atomic cell holding the threshold encryption key post DKG.
     pub(crate) threshold_dec_key: ThresholdKeyCell,
 
+    /// Chain id (namespace) for the sequencing chain.
+    pub(crate) namespace: ChainId,
+
     /// Chain configuration
-    pub(crate) chain_config: ChainConfig,
+    pub chain_config: ChainConfig,
```

Comment:
```suggestion
    pub(crate) chain_config: ChainConfig,
```

If necessary,

```rust
pub fn chain_config(&self) -> &ChainConfig {
    &self.chain_config
}
```
can be added.

Reply:
Agree. Changed it back now.
However, not a fan of the mix of getters and public fields like:
`let auction = Auction::new(cfg[0].0.chain_config().auction_contract.unwrap());`